src/openrct2/ride/water/RiverRapids.cpp

```diff
@@ -627,7 +627,7 @@ static void paint_river_rapids_track_right_quarter_turn_1_tile(
             PaintAddImageAsParent(session, imageId, { 0, 0, height }, { 28, 28, 11 }, { 4, 4, height });
 
             imageId = SPR_RIVER_RAPIDS_RIGHT_QUARTER_TURN_1_TILE_FRONT_SW_SE | session.TrackColours[SCHEME_TRACK];
-            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { 1, 1, 7 }, { 28, 28, height + 13 });
+            PaintAddImageAsParent(session, imageId, { 0, 0, height }, { 1, 1, 7 }, { 28, 28, height + 2 });
```

Comment:
Does this one need the +2?

Reply:
Yea, otherwise this happens 
![image](https://user-images.githubusercontent.com/1516893/171743543-5d2eb8bd-77b7-41fe-9fdd-6e4dfd3f677f.png)